app/controllers/catalog_controller.rb

```diff
@@ -265,24 +265,6 @@ def x_show
     tree_select
   end
 
-  # Tree node selected in explorer
-  def tree_select
-    @explorer   = true
-    @lastaction = "explorer"
-    self.x_active_tree = params[:tree] if params[:tree]
```

Comment:
@martinpovolny setting of self.x_active_tree is missing in https://github.com/ManageIQ/manageiq/pull/12320/files#diff-197a2eb3c02f33f29c3cd0ecc34082c4R142, is that ok?